dotcom-rendering/src/web/components/ArticleMeta.tsx

```diff
@@ -71,6 +71,12 @@ const stretchLines = css`
 	}
 `;
 
+const metaBorderTopStandfirstStyling = css`
```

Comment:
```suggestion
const borderColourWhenBackgroundRed = css`
```

This is a take it or leave it comment. This a hard thing to name!

Reply:
It's not always going to be red, is it? Isn't this for any liveblog?

Reply:
Good point!

Reply:
`borderColourWhenBackgroundDark`?